### ISSUE TITLE
Typo or missing info in xml_remove() man page?

### ISSUE THREAD

**Issue by maelle** [r-lib/xml2#280]
https://github.com/r-lib/xml2/blob/6fa5910008505f66aa50ab29bd8fc71a3ddf96e4/R/xml_modify.R#L9

It's especially upsetting since it sounds so frightening :wink:

**Comment by jimhester** [comment]
Agreed, basically we need to summarize part of https://xml2.r-lib.org/articles/modification.html#removing-nodes here I guess...

**Comment by maelle** [comment]
Or maybe this is a good use case for a re-usable Rmd piece. 😉
